00-programming-fundamentals/git-diff-pull-and-merge.md

```diff
@@ -126,10 +182,8 @@ Follow the link in the calendar to the Git Pull Activity, and follow the instruc
 ## Git Commands
 
 Here's a handy table of all the git commands we've learned so far.
-
 | Command             | Description |
 |---------------------|-------------|
-| `git init`          | Initialize an empty git repository
```

Comment:
Alas, poor `git init`!

I am curious why you decided to scrap this.

Reply:
We (almost) never ask them to run `git init` - they're always cloning something that we've made.